annet/adapters/netbox/models.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import List, Optional, Any
+
+from annet.annlib.netdev.views.hardware import HardwareView
+from .common_models import Entity, DeviceType, DeviceIp, IpAddress, Label
+
+
+@dataclass
+class Interface(Entity):
+    device: Entity
+    enabled: bool
+    display: str = ""
+    ip_addresses: List[IpAddress] = field(default_factory=list)
+
+
+@dataclass
+class Device(Entity):
```

Comment:
this collides with annet.storage.Device, we should either rename (preferably) or import it as `model.Device`